packages/data/addon/helpers/metric-format.js

```diff
@@ -13,6 +13,7 @@ import { hasParameters } from '../utils/metric';
 import { get } from '@ember/object';
 import { isPresent } from '@ember/utils';
 import Helper from '@ember/component/helper';
+import isObject from 'lodash/isObject';
```

Comment:
Do we need this or can we just use `typeof val === 'object'`

Reply:
`typeof val === 'object'` is pretty nasty if you get nulls and other objects that don't work like the plain ol js objects.

Reply:
or maybe `val === Object(val)` ?

Reply:
Still prefer _.isObject, since it is easier to read and understand

Reply:
@mattgraham1995 Actually need to use `isPlainObject` to exclude array case, so `val === Object(val)  ` will not work here